src/test/java/core/basesyntax/dto/TransactionDtoTest.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.dto;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class TransactionDtoTest {
```

Comment:
There is no point in testing this class. The DTO has no functionality whatsoever. It can only store data about an entity and you can read it out.